### PR TITLE
Run build on self hosted runner

### PR DESCRIPTION
**Pull request by arturkow2000**
Signed-off-by: Artur Kowalski <artur.kowalski@3mdeb.com>

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,11 +15,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-    container:
-      image: 3mdeb/coreboot-sdk:mkimage
```
Comment (macpijan) [inline .github/workflows/build.yml:20]: @arturkow2000 Do we really need to that? To call the `docker run` for each command? With local gitlab-runner we simply use the same job syntax (running the whole job in container). Each time I use GA currently, I feel that  the gitlab CICD is so much better IMHO.

Comment (arturkow2000) [reply .github/workflows/build.yml:20]: @macpijan GA detects that runner is inside Docker and fails
with `Error: Container feature is not supported when runner is already running inside container.`
https://github.com/3mdeb/coreboot/runs/2930800018?check_suite_focus=true#step:2:1

Comment (macpijan) [reply .github/workflows/build.yml:20]: This looks terrible, but might be the only workaround for now: https://github.com/actions/runner/issues/367

**Comment by macpijan** [comment]
picked to: https://github.com/3mdeb/coreboot/tree/talos_2_support_payload

**Pull request closed without merging**